example/sync.config.js

```diff
@@ -1,6 +1,6 @@
 export const SYNC_CONFIG = {
   // Set `enabled` to `true` to enable sync.
-  enabled: false,
+  enabled: true,
   // Add your Realm App ID here if sync is enabled.
-  appId: '<Your App ID>',
+  appId: 'realmtemplate-waxdf',
```

Comment:
🙃

Reply:
Everytime 😅

Reply:
git update-index --assume-unchanged [<file> ...]

Reply:
Perhaps it is worth modifying this to use an environment variable (and then e.g. have a `.env` file with your app id in which can be `gitignore`d)?

Reply:
Done.  It's probably best practice to inject your app id through an env var anyway.